infra/modules/providers/azure/app-service/output.tf

```diff
@@ -7,3 +7,13 @@ output "app_service_ids" {
   description = "The resource ids of the app service created"
   value       = ["${azurerm_app_service.appsvc.*.id}"]
 }
+
+output "app_service_identity_tenant_id" {
+  description = " The Tenant ID for the Service Principal associated with the Managed Service Identity of this App Service."
+  value       = "${azurerm_app_service.appsvc.0.identity.0.tenant_id}"
```

Comment:
We can avoid using a hard-coded index here by using a wildcard match (i think). Is there a reason we need the 0 index?

`${azurerm_app_service.appsvc.*.identity.*.tenant_id}`

https://github.com/hashicorp/terraform/issues/18744

Reply:
This was by design. We want to return a single tenant id as there shouldn't be a scenario where you provision a fleet of app services across multiple tenants.